README.md

```diff
@@ -1 +1,116 @@
-# To-Do-List
+# :green_book: Table of Contents
+
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Live-Demo](#live-demo)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# :book: [Awesome Books]
+
+**[To-Do List]** is a microverse project where we build a basic website that allows users to add/remove list. We achieved that by using JavaScript objects and arrays. We also dynamically modified the DOM and add basic events.
+
+
+
+## :hammer_and_wrench: Built With
+
+### Tech Stack
+
+**Client**
+HTML, CSS and Javascript
+
+### Key Features
+
+- **[- Use Javascript, HTML & CSS linters]**
+- **[- Build a basic website that allows users to add/remove task list.]**
+- **[- Used JavaScript objects and arrays.]**
+- **[- Dynamically modified the DOM and add basic events.]**
+
+## :computer: Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a web browser, node, npm and an IDE like VS Code should be installed in your computer.
+
+### Setup
+
+Open Terminal & Clone this repository to your desired folder:
+
+```sh
+cd <path-to-the-folder>
+git clone https://github.com/tasnimchy96/to-do-list.github.io
+cd to-do-list.github.io
+code .
+```
+### Install
+
+Install this project using the cloned repo or download and use the files to your local machine.
+
+```sh
+npm install
+```
+### Usage
+
+```usage
+- To run the project open the index.html file in your web browser or you can also open live server from the IDE.💯
+```
+### Run tests
+
+```test
+npx hint .
+npx eslint .
+```
+
+### :rocket: Live-Demo
+
+- [Live-Demo](https://tasnimchy96.github.io/to-do-list.github.io/)
```

Comment:
[OPTIONAL] Please fix your live link because it's not working if you didn't deploy your project yet then please write here `Coming Soon` instead of adding the link :slightly_smiling_face: